ops/cmd/print_staging_report/main.go

```diff
@@ -20,10 +21,15 @@ import (
 )
 
 var (
-	L1RPCURLFlag = &cli.StringFlag{
-		Name:     "l1-rpc-url",
-		Usage:    "The URL of the L1 RPC endpoint.",
-		EnvVars:  []string{"L1_RPC_URL"},
+	SepoliaRPCURLFlag = &cli.StringFlag{
+		Name:    "sepolia-rpc-url",
+		Usage:   "The URL of the Sepolia RPC endpoint.",
+		EnvVars: []string{"SEPOLIA_RPC_URL"},
+	}
+	MainnetRPCURLFlag = &cli.StringFlag{
+		Name:     "mainnet-rpc-url",
+		Usage:    "The URL of the mainnet RPC endpoint.",
+		EnvVars:  []string{"MAINNET_RPC_URL"},
 		Required: true,
```

Comment:
Why make these separate instead of using the `L1RPCURLFlag` as before? Also curious why `MainnetRPCURLFlag` is required but `SepoliaRPCURLFlag` is not. Are you expecting that a testnet will never be added to the registry by itself? Instead, it must always be accompanied by its mainnet when entering the registry?

Reply:
Both flags should be required. The flags are separated due to CircleCI's env var design. It's easier to load both into CircleCI as env vars and have the tool choose which one to use than it is to have the CircleCI bash script in the job pick the right URL.

Reply:
fixed the required flag